webroot/results/p.php

```diff
@@ -1,6 +0,0 @@
-<?php
-
-if(!isset($_REQUEST['personId']) && isset($_REQUEST['i']))
-  $_REQUEST['personId'] = $_REQUEST['i'];
```

Comment:
Ah, this makes it look like your redirect over in #1579 is a little wrong, since your nginx redirect assumes the parameter is `i`, whereas this allows for it to be `personId` if it is set.